content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -16,79 +16,96 @@ import Tooltip from "shared/components/ui/Tooltip";
-        "autoscaling:DescribeAutoScalingGroups",
-        "autoscaling:DescribeAutoScalingInstances",
-        "autoscaling:DescribeLaunchConfigurations",
-        "autoscaling:DescribeTags",
-        "ec2:DescribeInstanceTypes",
-        "ec2:DescribeLaunchTemplateVersions"
-      ],
-      "Resource": ["*"]
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:SetDesiredCapacity",
-        "autoscaling:TerminateInstanceInAutoScalingGroup",
-        "ec2:DescribeInstanceTypes",
-        "eks:DescribeNodegroup"
-      ],
-      "Resource": ["*"]
-    }
-  ]
-}
-```
-<InfoBox>
-
-To deploy AWS Autoscaler, update the Kubernetes pack `managedMachinePool` node group with the **ARN** of the autoscaler policy created for the service account.
-
-```yaml
-managedMachinePool:
-  #roleName: {{ name of the self-managed role | format "${string}" }}
-
-  ## A list of additional policies to attach to the node group role
-  roleAdditionalPolicies:
-  - "arn:aws:iam::012345678910:policy/autoscalingpolicy"
-```
-</InfoBox>
-
-
-## Versions Supported
+# Versions Supported
 
 <Tabs>
 
-<Tabs.TabPane tab="1.22.x" key="1.22.x">
+<Tabs.TabPane tab="1.24+" key="1.24.x">
+
+**1.24+**
+
+You can use AWS cluster autoscaler with the Kubernetes control plane (previously referred to as *master*) version 1.24.x or higher. 
+
 
-**1.22.2**
 
-</Tabs.TabPane>
 
-<Tabs.TabPane tab="1.0.x" key="1.0.x">
+# Prerequisite
+
+* Kubernetes version 1.24.x and above.
+
+
+* A full cluster autoscaler policy applied to the cluster in AWS console as an inline policy. Below is the recommended full cluster autoscaler policy.
+<br />
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:DescribeAutoScalingGroups",
+          "autoscaling:DescribeAutoScalingInstances",
+          "autoscaling:DescribeLaunchConfigurations",
+          "autoscaling:DescribeScalingActivities",
+          "autoscaling:DescribeTags",
+          "ec2:DescribeInstanceTypes",
+          "ec2:DescribeLaunchTemplateVersions"
+        ],
+        "Resource": ["*"]
+      },
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:SetDesiredCapacity",
+          "autoscaling:TerminateInstanceInAutoScalingGroup",
+          "ec2:DescribeImages",
+          "ec2:GetInstanceTypesFromInstanceRequirements",
+          "eks:DescribeNodegroup"
+        ],
+        "Resource": ["*"]
+      }
+    ]
+  }
+  ```
+
+
+* Update the Kubernetes pack's  `managedMachinePool` node group with the **ARN** of the autoscaler policy created for the service account. Here is an example YAML configuration. 
+<br />
+
+  ```yaml
+  managedMachinePool:
+    #roleName: {{ name of the self-managed role | format "${string}" }}
+
+    ## A list of additional policies to attach to the node group role
+    roleAdditionalPolicies:
+    - "arn:aws:iam::012345678910:policy/autoscalingpolicy"
+  ```
+
+## Usage

```

Comment:
This Usage section might be all you need for this pack, unless Engineering has additional information.